### DISCUSSION TITLE
Assistance Required for Crazyflie: Unstable Flight

### DISCUSSION THREAD

**Discussion by gurpreet-98** [bitcraze/discussions#1371] [Q&A]
I am writing to seek assistance regarding an issue encountered while flying crazyflie using Client software on windows 11. Based on the instruction provided on Bitcraze site I was able to install Python , pip package and git as well (version :picture attached below)
 
 
![image](https://github.com/bitcraze/discussions/assets/80755398/96aa026a-43ae-4089-b4a1-ffe7576c944e)

 
Further, installed cfclient using the following source code.
 
To modify and examine the code also installed clib using following commands.
git clone git clone https://github.com/bitcraze/crazyflie-lib-python
cd crazyflie-lib-python
pip install .
 
Cloned the whole Bitcraze project using .cmd on windows.
 
git clone https://github.com/bitcraze/crazyflie-clients-python
cd crazyflie-clients-python
Install the client from source.
pip3 install -e .
 
After following the above steps mentioned I was able to open cfclient, however there were few tabs which I could not operate on such as was not able to change the assist mode to Hover and could not specify Anchor status mode change under Loco Positioning Tab. 
 
 
![image](https://github.com/bitcraze/discussions/assets/80755398/994d2378-83e5-4558-8bdf-f398a7f8a09a)

 
Before operating on Crazyflie, updated the latest firmware (cf2) by using the bootloader mode. Configured, calibrated, and estimated the geometry of the lighthouse positioning system based on the steps mentioned on site. 
 
Now after installing and configuring everything whenever I try to operate on crazyflie it appears unstable and does not respond accurately to control inputs from client software and sometimes it drifts uncontrollably. 
 
Even implemented the motion_commander_demo.py ( without using LHS system) file to see if operates fine but eventually the behavior does not reflect any changes. There is no specific error message displayed on the client software, but the behavior of crazyflie suggest that there is something not functioning properly. Please let me know if I am missing some steps in the installation process or any troubleshooting steps or diagnostics that I can perform to identify the root cause of the fight instability. I have also attached an video running motion_commander_demo code on Crazyflie. 

https://github.com/bitcraze/discussions/assets/80755398/6c61590c-585f-4d52-969e-2a2d35f35cc2

The above video shows the behavior of Crazyflie without using Lighthouse System. Even after configuring the LHS system the crazyflie still behaves weirdly during flight. 
 
I appreciate any assistance you can provide to help me resolve this issue. Thank you for your support!

**Comment by knmcguire** [comment]
Just something about the tabs:  The loco positioning system is not supposed to be fully functional if you are not connected with a Crazyflie and a loco positioning system. That is normal behavior of the cfclient

Without the lighthouse system, this behavior is normal. The Crazyflie is supposed to get a position estimate and if it doesn't receive information to infer this state, it won't be able to fly stable. That is normal.

You are mentioning that it does the same with the LH system enabled. In that case there are some questions:
- Do you see a green LED on the LH Deck on?
- Is the positioning stable of the Crazyflie in the lighthouse tab?
- Where you able to complete the [full lighthouse tutorial ](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/)without problems, and if not, clearly indicate at which step it went wrong.
